### PR TITLE
updated feedtime validation

### PR DESCRIPTION
**Pull request by Neiftly**
changed the validator to include not allowing future dates and then did two tests.

### CODE REVIEW

File: src/test/java/edu/kcc/animal/AnimalTest.java [hunk 1]
```diff
@@ -268,13 +268,25 @@ public void testSetLastFeedingTime() {
     // TODO
     @Test
     public void testSetLastFeedingTimeMoreThan2DaysAgoBad() {
-        fail("The test case is a prototype.");
+        LocalDateTime ldtThreeDays = LocalDateTime.now().minusDays(3);
+        Animal instance = new Animal();
+        try {
+            instance.setLastFeedingTime(ldtThreeDays);
+        } catch (Exception ex) {
+            System.out.println(ex);
```
Comment (mlhaus) [inline src/test/java/edu/kcc/animal/AnimalTest.java:276]: You'll want to do something like `assertTrue(true);` here.

File: src/test/java/edu/kcc/animal/AnimalTest.java [hunk 2]
```diff
@@ -268,13 +268,25 @@ public void testSetLastFeedingTime() {
...
     // TODO
     @Test
     public void testSetLastFeedingTimeFutureDateBad() {
-        fail("The test case is a prototype.");
+        LocalDateTime ldtFutureDate = LocalDateTime.now().plusDays(1);
+        Animal instance = new Animal();
+        try {
+        instance.setLastFeedingTime(ldtFutureDate);
+        } catch(Exception ex) {
+            System.out.println(ex);
```
Comment (mlhaus) [inline src/test/java/edu/kcc/animal/AnimalTest.java:288]: You need to include an assert statement here.

**Review by mlhaus — CHANGES_REQUESTED** [review]
Missing assert statements

**Comment by Neiftly** [comment]
I think I have updated the pull request with new commits.

**Pull request merged**